src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs

```diff
@@ -62,6 +65,24 @@ public static ValueTask<TypedData> ToRpc(this object value, ILogger logger, Grpc
             }
         }
 
+        internal static TypedData ToBindingData(this ParameterBindingData data, ILogger logger)
+        {
+            var bindingData = new BindingData
+            {
+                Version = data.Version,
+                ContentType = data.ContentType,
+                Source = data.Source,
+                Content = data.Content.ToString()
```

Comment:
Do you expect the conversion here to change once we update the content to be `BinaryData`? Are we going to keep the proto definition as a string?

Reply:
We can do string or bytes - there aren't any other types that fit. If we use bytes, we can do this `ByteString.CopyFrom(data.Content)`

Reply:
@fabiocav Currently using bytes, let me know what you think